src/main/java/teammates/common/util/Config.java

```diff
@@ -25,6 +25,21 @@ public final class Config {
     /** The value of the "app.frontend.url" in build.properties file. */
     public static final String APP_FRONTEND_URL;
 
+    /** The value of the "app.postgres.host" in build.properties file. */
+    public static final String POSTGRES_HOST;
```

Comment:
I think we can standardise the property to either not be prefixed by "app-" (like here) or prefix the property with "app-" like in your `src/main/resources/build-dev.template.properties`

On a side note would you know what "app-" is? I'm wondering if it means its a property of GAE, else we probably drop the "app" from postgres attributes since its on Cloud SQL

Reply:
I agree the lack of standardization is quite strange too (I flip flopped between both a few times)

For the variable name, I've prefixed without "APP" to be more in-line with other variables for other services (E.g `EMAIL_SENDEREMAIL`, `MAILGUN_APIKEY`). The only variables with "APP" prefix seem to for truly "application" details, like `APP_FRONTEND_URL`, `APP_ADMINS`.

For the property prefix, every single property has the 'app' prefix. I believe it's a matter of standardization only, and it doesn't contribute to functionality.

Since it affects many other variables, should we handle this in a separate PR?

Reply:
@NicolasCwy Maybe can open an new issue (which can be labelled as good first issue) for this? Let's merge this in first.

Reply:
Yup was just thinking of changing this one property, but I'll create an issue anyway